lib/workers/repository/index.ts

```diff
@@ -91,11 +93,16 @@ export async function renovateRepository(
         repoResult = processResult(config, errorRes);
       }
 
-      return { config, localDir };
+      return { config, localDir, errorRes };
     },
   );
 
   try {
+    // only continue if init stage was successful
+    if (errorRes) {
+      throw new Error(errorRes);
+    }
```

Comment:
This code doesn't exist in the code before #38607 - what was the thinking here? (Or have I missed it in the existing code)?